pint/toa.py

```diff
@@ -1,12 +1,12 @@
-from __future__ import absolute_import, print_function, division, unicode_literals
+from __future__ import absolute_import, print_function, division
 import re, sys, os, numpy, gzip, copy
 from . import utils
 from .observatory import Observatory, get_observatory
 from .observatory.topo_obs import TopoObs
 from . import erfautils
 import astropy.time as time
 from . import pulsar_mjd
-from six.moves import cPickle as pickle
+from astropy.extern.six.moves import cPickle as pickle
```

Comment:
Do not use `astropy.extern.six` it is deprecated. We already depend on `six` directly so just import that.

Reply:
Fixed